src/Exception/RuntimeException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace PhpTuf\ComposerStager\Exception;
+
+class RuntimeException extends \RuntimeException
```

Comment:
This seems like a little too much abstraction :) What's the rationale here?

Reply:
Two things:

1. To give client code a single exception family to work with--so that it could, for example, catch `PhpTuf\ComposerStager\Exception` and know it would be catching any and only Composer Stager exceptions.
2. To keep a clean layered architecture, so that client code only has to know about Composer Stager itself, never the layer underneath it (i.e., Symfony) and depend on _its_ technical details. In other words, to isolate coupling on unstable dependencies.

Reply:
I don't feel super strongly about this, to be honest, but IMHO those arguments make much more sense in the context of this being an API that other code will consume. That seems unlikely; this is much more of a self-contained command that will be used by developers, but not necessarily integrated with at the code level. I could be wrong here, but that's why I wonder if this level of abstraction is necessary.

Of course, by the very same token...if people DO end up using it as a command and not as a library (which I suspect will be the case), then it'll be trivial to remove this exception later if some trimming feels necessary. Thus, I don't feel especially strongly about it. Maybe just a doc comment explaining that this is a base exception for run-time errors raised by the execution of the stager command, is all that's needed.

Reply:
Actually, this whole application is being _explicitly_ designed to be callable via its PHP API. Drupal core, for example, won't want to create a new process to invoke `composer-stage stage`. It'll want to call `Stager::stage()` directly in-process.